### PR TITLE
Abilities: Minor changes & grooming

### PR DESCRIPTION
**Pull request by differentreality**
Venue has show action, not index.
Make the ability to schedule an Event more clear.

### CODE REVIEW

File: app/models/ability.rb [hunk 1]
```diff
@@ -127,6 +127,7 @@ def signed_in_with_organizer_role(user)
     can :manage, Vday, conference_id: conf_ids_for_organizer
     can :manage, CallForPaper, conference_id: conf_ids_for_organizer
     can :manage, Event, conference_id: conf_ids_for_organizer
+    can :schedule, Event, conference_id: conf_ids_for_organizer
```
Comment (hennevogel) [inline app/models/ability.rb]: `:manage` means everything

Comment (differentreality) [inline app/models/ability.rb]: Yes indeed, and in fact there does not even exist a function called 'schedule' in the controller. 
The reason I thought we could do it like this is merely for clarity.
You can schedule an event if you can update its start time. Perhaps that is not 100% clear to everyone.
If you think it does not really add any value, I ll remove it.

Comment (hennevogel) [inline app/models/ability.rb]: It doesn't. Either you explicitly allow only specific methods and remove `:manage` or you use `:manage`. Everything else is just misleading IMHO

**Comment by differentreality** [comment]
Ready for review!

**Comment by ChrisBr** [comment]
LGTM :+1:

**Pull request merged**